openedx/openedx-platform
# [ENT-491] Remove SSO consent features; verify fully-populated consent URL generation

This pull request removes logic related to the Enterprise SSO hooks to get in the middle of the third-party auth pipeline.

**JIRA tickets**: [ENT-491](https://openedx.atlassian.net/browse/ENT-491)

**Dependencies**: edx/edx-enterprise#152

**Sandbox URL**: https://pr15624.sandbox.opencraft.hosting/

**Merge deadline**: 31 July 2017

**Testing instructions**:

1. Verify that you can no longer configure an EnterpriseCustomer with "at login" or "optional" consent requirements.
2. Verify that the third-party auth pipeline no longer includes any Enterprise items.

**Settings**
```yaml
EDXAPP_FEATURES:
  ENABLE_COMBINED_LOGIN_REGISTRATION: true
```
